wiki/osu!supporter/en.md

```diff
@@ -1,61 +1,146 @@
 
-The usercard displayed above the slider is the user you are buying osu!supporter for. By default, it should be your usercard. To gift to a friend, enter your friend's username (not user ID) into the grey box below the usercard. Once this is done, the usercard will change to match the username you had entered. From here, you can then select the duration of osu!supporter you want to gift, but keep in mind that there is no promotion for group gifting.
+### osu!direct
 
-You can only buy or gift osu!supporter to one user at a time within this screen. To gift another user, you will need to go back to the [osu!supporter store page](https://osu.ppy.sh/store/products/supporter-tag) and repeat the steps for adding another user.
+osu!direct is the in-game beatmap search and download engine available to the supporters. It may be accessed from the main menu by clicking the vertical `osu!direct` label on the left side of the screen. In addition to the built-in beatmap listing, osu!direct provides other ways to obtain new beatmaps without leaving the game:
 
-### Checking out
+- In-chat links, which point to beatmaps on the osu! website, now trigger the download prompt in-game;
+- [Spectators](/wiki/Gameplay/Spectating) with osu!supporter are able to automatically download beatmaps their target is playing;
+- Players with osu!supporter may opt-in to automatically linking the beatmap they are currently playing in the `#spectator` chat tab.
 
-Once you are done, click `Add to Cart` to put it into the osu!store shopping cart.
+### Extended leaderboards
 
-![osu!store shopping cart](img/shopping-cart.jpg?1 "osu!store shopping cart with osu!supporter for flyte")
+Supporters have access to several beatmap-specific [leaderboards](/wiki/Beatmap#leaderboards) available both in-game and on the website:
 
-When you are ready to make the purchase, go to your [shopping cart](https://osu.ppy.sh/store/cart) and click `Checkout`. Then follow the prompts to make your payment.
+- Global ranking for every combination of [game modifiers](/wiki/Game_modifier);
+- Country-specific ranking made of players bearing the same flag;
+- Friend ranking that shows a player's position on the map compared to their friends.
 
-## Conclusion
+### Increased limits
 
-After the transaction is completed, the Recent Activity section in your account (if you had brought it your self) or the accounts you have gifted will say:
+osu! offers more relaxed limits on different things to the supporters:
+
+| Quota type | Regular limit | osu!supporter limit |
+| :-- | :-: | :-: |
+| Uploaded beatmaps | 4 + `min(ranked beatmaps, 2)` | 8 + `min(ranked beatmaps, 4)` |
+| Online beatmap favourites | 100 | 1000 |
+| Friend list length | 250 | 500 |
+
+Additionally, the osu!supporter bearers are granted more lenient download speed limits, as well as the limits on amount of beatmaps downloaded per hour.
```

Comment:
while "osu!supporter" is treated as a noun here, it doesn't look correct (as a verb it would be something like "supporter-bearer")

```suggestion
Additionally, bearers of osu!supporter are granted more lenient beatmap download limits.
```